cirq/ops/common_gates.py

```diff
@@ -689,6 +712,29 @@ def _pauli_expansion_(self) -> value.LinearDict[str]:
     def _phase_by_(self, phase_turns, qubit_index):
         return self
 
+    def controlled(self,
+                   num_controls: int = None,
+                   control_values: Optional[Sequence[
+                       Union[int, Collection[int]]]] = None,
+                   control_qid_shape: Optional[Tuple[int, ...]] = None
+                  ) -> raw_types.Gate:
+        """
+        Specialize controlled for CZPow to return corresponding controlled
+        CCZPow when the last control (which acts first semantically) is a
+        default-type control qubit.
```

Comment:
I think I would also revise this pydoc.  Again, I think that we should use the actual class names (CZPowGate, etc) instead of abbreviations in our docs.

Same for the other pydocs below.

Reply:
Done